### DISCUSSION TITLE
Wiiiide website

### DISCUSSION THREAD

**Discussion by Ryan365Apps** [TanStack/table#3942] [Ideas] [answered]
Hi,
Firstly, thanks for making React Table - it's awesome. Superb, outstanding. 
Secondly, the [website](https://react-table.tanstack.com/docs/examples/kitchen-sink) is super useful but it maxes out at max-wdith:1400.
My screen is wiiiiiiiiide. super wide. Wider than the gap between my ambition and skill, wider than the valley between my hopes and dreams. It's wide. 
I can't see all the goodness of the examples with the CodeSandbox without poking around in dev tools. 
Anyway, just saying. Have a nice day.

**Comment by tannerlinsley** [comment, marked as answer]
A new website is in the works. Stay tuned.

Tanner Linsley
